code/_globalvars/bitfields.dm

```diff
@@ -341,7 +341,9 @@ GLOBAL_LIST_INIT(bitfields, list(
 		"HAS_NO_HAIR" = HAS_NO_HAIR,
 		"IS_PLANT" = IS_PLANT,
 		"IS_SYNTHETIC" = IS_SYNTHETIC,
-		"NO_STAMINA" = NO_STAMINA
+		"NO_STAMINA" = NO_STAMINA,
+		"IS_SECTOID_ALIEN" = IS_SECTOID_ALIEN,
```

Comment:
Check the `#define ismoth(H)` and make one similar instead.
These flags are for generic species traits.

Reply:
I want to add a second race that can use their guns at some point, i'll rename the flag though